README.md

```diff
@@ -1,103 +1,151 @@
+[![Crates.io](https://img.shields.io/crates/v/ntpd.svg)](https://crates.io/crates/ntpd)
 
 
 # ntpd-rs
 
-ntpd-rs is an NTP implementation written in Rust, with a focus on security and stability. It includes client and server functionality and supports NTS.
+ntpd-rs is an NTP implementation written in Rust, with a focus on security and
+stability. It includes client and server functionality and supports NTS.
 
 If a feature you need is missing please let us know by opening an issue.
 
 ## Installation
 
-The recommended way of installing ntpd-rs is with the pre-built packages from the [releases](https://github.com/pendulum-project/ntpd-rs/releases) page. The installers automatically handle setting up users, permissions and configuration.
-
-Alternatively, you can use `cargo install ntpd` or build from source.
+The recommended way of installing ntpd-rs is with the pre-built packages from
+the [releases page]. The installers automatically handle setting up users,
+permissions and configuration. Alternatively, you can use `cargo install ntpd`
+or build from source by cloning the repository yourself.
 
 ### Build from source
 
-Currently, ntpd-rs only supports Linux-based operating systems. Our current testing only targets Linux kernels after version 5.0.0, older kernels may work but this is not guaranteed.
+Currently, ntpd-rs only supports Linux-based operating systems. Our current
+testing only targets Linux kernels after version 5.0.0, older kernels may work
+but this is not guaranteed.
 
-ntpd-rs is written in rust. We strongly recommend using [rustup](https://rustup.rs) to install a rust toolchain, because the version provided by system package managers tends to be out of date. Be sure to use a recent version of the rust compiler.
+ntpd-rs is written in rust. We strongly recommend using [rustup] to install a
+rust toolchain, because the version provided by system package managers tends to
+be out of date. Be sure to use a recent version of the rust compiler. To build
+ntpd-rs run
 
-To build ntpd-rs run
 ```sh
 cargo build --release
 ```
-This produces a `ntp-daemon` binary at `target/release/ntp-daemon`, which is the main NTP daemon.
 
-Before running the ntpd-rs daemon, make sure that no other NTP daemons are running. E.g. when chrony is running
+This produces a `ntp-daemon` binary at `target/release/ntp-daemon`, which is the
+main NTP daemon. Before running the ntpd-rs daemon, make sure that no other NTP
+daemons are running. E.g. when chrony is running
+
 ```sh
 systemctl stop chronyd
 ```
 
-The ntpd-rs daemon requires elevated permissions to change the system clock. It can be tested against servers in the [NTP pool](https://ntppool.org):
+The ntpd-rs daemon requires elevated permissions to change the system clock.
+
 ```sh
-sudo ./target/release/ntp-daemon -p pool.ntp.org -p pool.ntp.org -p pool.ntp.org -p pool.ntp.org
+sudo ./target/release/ntp-daemon
```

Comment:
this will now (try to) use the `ntp.toml` at the default location. It is not guaranteed that this file exists, so it is quite likely that someone following these instructions now runs into a panic

Reply:
I'll add the `-c` option with the example config file in the root of the project to this example to make sure people don't run into that specific error.